addons/web_editor/static/src/js/wysiwyg/wysiwyg_iframe.js

```diff
@@ -7,7 +7,6 @@ var core = require('web.core');
 var config = require('web.config');
 
 var qweb = core.qweb;
-var promiseCommon;
```

Comment:
Why is this not used anymore? Make sure it's normal.

Reply:
It's not used in master.